Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2_test.py

```diff
@@ -1083,3 +1088,400 @@ def test_update_alerts_command(args, expected_error, mcafeensmv2_client):
     with pytest.raises(Exception) as e:
         update_alerts_command(mcafeensmv2_client, args)
         assert expected_error == str(e.value)
+
+
+@pytest.mark.parametrize('input, output', [(777, {'method': 'GET', 'url_suffix': '/domain/9/policyassignments/device/777'}),
+                                           (None, {'method': 'GET', 'url_suffix':
+                                                   '/domain/9/policyassignments/device'})])
+def test_list_device_policy_request__with_and_without_device_id(mocker, mcafeensmv2_client, input, output):
+    """
+    Given:
+        - A device id or no device id.
+    When:
+        - nsm-list-device-policy command is executed.
+    Then:
+        - The http request is called with the right arguments.
+    """
+    from McAfeeNSMv2 import Client
+    http_request = mocker.patch.object(mcafeensmv2_client, '_http_request')
+    Client.list_device_policy_request(mcafeensmv2_client, domain_id=9, device_id=input)
```

Comment:
Why do we need to import the Client in each test instead of using: 
`mcafeensmv2_client.list_device_policy_request(domain_id=9, device_id=input)` for example. 

Maybe I missed something... just wondered.